src/content/app/genome-browser/hooks/useGenomeBrowserAnalytics.ts

```diff
@@ -0,0 +1,303 @@
+      action: 'track_name_' + (isTrackNameShown ? 'on' : 'off')
+    });
+  };
+
+  const trackFeatureLabelToggle = (
+    selectedCog: string,
+    areFeatureLabelsShown: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'feature_labels_' + (areFeatureLabelsShown ? 'on' : 'off')
+    });
+  };
+
+  const trackShowSeveralTranscriptsToggle = (
+    selectedCog: string,
+    isSeveralTranscriptsShown: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action:
+        'several_transcripts_' + (isSeveralTranscriptsShown ? 'on' : 'off')
+    });
+  };
+
+  const trackShowTranscriptsIdToggle = (
+    selectedCog: string,
+    shouldShowTranscriptIds: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'transcript_labels_' + (shouldShowTranscriptIds ? 'on' : 'off')
+    });
+  };
+
+  const trackApplyToAllInTackSettings = (
+    selectedCog: string,
+    shouldApplyToAll: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'apply_to_all - ' + (shouldApplyToAll ? 'unselected' : 'selected')
+    });
+  };
+
+  const trackTrackPanelTabChange = (selectedTrackPanelTab: TrackSet) => {
+    sendTrackEvent({
+      category: 'track_panel_tab',
+      label: selectedTrackPanelTab,
+      action: 'selected'
+    });
+  };
+
+  const trackTrackPanelSectionToggled = (sectionName: string) => {
+    sendTrackEvent({
+      category: 'track_panel',
+      label: sectionName,
+      action: 'section_toggled'
+    });
+  };
+
+  const trackResetIcon = () => {
+    sendTrackEvent({
+      category: 'reset_icon',
+      action: 'clicked'
+    });
+  };
+
+  const trackSidebarModalViewToggle = (view: string) => {
+    sendTrackEvent({
+      category: 'sidebar_toolstrip',
+      action: 'modal_opened',
+      label: view
+    });
+  };
+
+  const trackDrawerOpened = (drawerView: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'opened',
+      label: drawerView
+    });
+  };
+
+  const trackDrawerSequenceViewed = (sequenceType: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_viewed',
+      label: sequenceType
+    });
+  };
+
+  const trackDrawerSequenceCopied = (sequenceType: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_copied',
+      label: sequenceType
+    });
+  };
+
+  const trackDrawerSequenceDownloaded = (selectedOptions: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_downloaded',
+      label: selectedOptions
+    });
+  };
+
+  const trackSpeciesChangeLinkClicked = () => {
+    sendTrackEvent({
+      category: 'app_bar',
+      action: 'change_link_clicked'
+    });
+  };
+
+  const trackAppBarGenomeChanged = () => {
```

Comment:
Why do you need the word "appBar" here? There's no other place for changing a genome.

Reply:
True. I've updated it